railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+* Use app namespace instead of rails namespace from the framework rake tasks.
```

Comment:
Needs more info. Mention some of the tasks and that the `rails:` namespace has been deprecated.